src/runtime/debug/debug.go

```diff
@@ -58,3 +66,60 @@ type Module struct {
 func SetGCPercent(n int) int {
 	return n
 }
+
+// Start of stolen from big go. TODO: import/reuse without copy pasta.
```

Comment:
I do believe a copyright notice should be copied here...?

Reply:
isn’t this covered by https://github.com/tinygo-org/tinygo/blob/release/LICENSE#L3 ?

Reply:
I think the copyright folks like to have a greppable marker in each file.

I like the way runtime/timer.go does it: copy the three top llines from the file you're copying, and change "Copyright" to "Portions copyright".

Reply:
Thanks, I was looking for an example, thanks for mentioning src/runtime/timer.go

Done